database/models/messages.db.php

```diff
@@ -55,6 +56,9 @@ class Conversation
     #[OneToMany(mappedBy: "conversation", targetEntity: Message::class)]
     public Collection $messages;
 
+    #[OneToOne(targetEntity: Event::class, mappedBy: 'conversation', cascade: ["remove"])]
+    public Event $event;
```

Comment:
I think this is incorrect: a conversation doesn't have to be related to an event, it can be related to something else

Reply:
Right, I will use the [one to one unidirectional](http://doctrine-project.org/projects/doctrine-orm/en/3.5/reference/association-mapping.html#one-to-one-unidirectional) association type !